jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java

```diff
@@ -154,6 +154,8 @@ public EventCollection newCollection() {
         if (clientId != null) {
             configs.put(ProducerConfig.CLIENT_ID_CONFIG, clientId);
         }
+        configs.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, 
+                    System.getProperty("org.kie.jbpm.event.emitters.kafka.max.block.ms", "2000"));
```

Comment:
Shouldn't we leave to the default `ProducerConfig.MAX_BLOCK_MS_CONFIG` value (60secs)? - wdyt?

Reply:
Problem is that default one is too long, thats the point of this JIRA ;)

Reply:
Right, it keeps on waiting (by default 60 sec) if broker is down and cannot establish a socket connection. Value can be overridden by each customer by means of the system variable.